app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
@niranjan94 
Date and time entered and timezone selected :
```
"starts-at":"2017-07-12T11:22:33+05:30",
"ends-at":"2017-07-16T11:22:33+05:30",
"timezone": "Asia/Kolkata",
```
When `changeToTimezone` function is used ---

Response is :
```
"timezone": "Asia/Kolkata",
"starts-at": "2017-07-12T05:52:33+05:30",
"ends-at": "2017-07-16T05:52:33+05:30"
```

When removed `changeToTimezone` function ---
```
"timezone": "Asia/Kolkata",
"starts-at": "2017-07-12T05:52:33+00:00",
"ends-at": "2017-07-16T05:52:33+00:00"
```

Datetime in db ----

`starts_at : 2017-07-12 05:52:33+00`
`ends_at : 2017-07-16 05:52:33+00`